bin/share

```diff
@@ -1,175 +1,7 @@
 #!/usr/bin/env python
```

Comment:
So `bin/share` is now just an alias for `sharectl` (or they're both aliases for the same thing), is that right?